apps/cookbook/src/app/showcase/dropdown-showcase/dropdown-showcase.component.html

```diff
@@ -1,65 +1,82 @@
+<p>
+  Dropdown displays multiple choices for user selection and appears upon interaction with an
+  elemeent.
+</p>
 <cookbook-dropdown-example-configuration
   [size]="size"
   (sizeChanged)="setSize($event)"
 ></cookbook-dropdown-example-configuration>
 <h2>Simple</h2>
 <h3>Default</h3>
 <cookbook-example-viewer [html]="defaultExample.template">
-  <div>
+  <div class="example-frame">
     <cookbook-dropdown-example-default
       [size]="size"
       #defaultExample
     ></cookbook-dropdown-example-default>
   </div>
 </cookbook-example-viewer>
 <h3>Scroll</h3>
+<p>The dropdown becomes scrollable if it contains more than 8 items.</p>
 <cookbook-example-viewer [html]="scrollExample.template">
-  <div>
+  <div class="example-frame">
     <cookbook-dropdown-example-scroll
       [size]="size"
       #scrollExample
     ></cookbook-dropdown-example-scroll>
   </div>
 </cookbook-example-viewer>
 <h3>Pre-selected</h3>
+<p>Use this if you want an initial preselected item.</p>
 <cookbook-example-viewer [html]="preSelectedExample.template">
-  <div>
+  <div class="example-frame">
     <cookbook-dropdown-example-pre-selected
       [size]="size"
       #preSelectedExample
     ></cookbook-dropdown-example-pre-selected>
   </div>
 </cookbook-example-viewer>
 <h3>Block level</h3>
+<p>
+  To display the dropdown in full width of its container, apply block-level styling to the dropdown.
+</p>
 <cookbook-example-viewer [html]="expandExample.template">
-  <div>
+  <div class="example-frame">
     <cookbook-dropdown-example-expand
       [size]="size"
       #expandExample
     ></cookbook-dropdown-example-expand>
   </div>
 </cookbook-example-viewer>
 <h3>Right aligned</h3>
+<p>
+  The dropdown can be right aligned, to place the dropdown with different alignment than the default
+  left alignment.
+</p>
```

Comment:
Consider being a bit more specific here.
```suggestion
<p>
  The dropdown can be right aligned, so the list of items appear along the right-most edge of the trigger button.
</p>
```